frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/data-resiliency/data-resiliency.tsx

```diff
@@ -19,20 +19,20 @@ const getCapacityStats = (response) => {
   return _.get(response, 'data.result[0].value[1]');
 };
 
-const DataResiliencyStatusBody: React.FC<DataResiliencyStatusBody> = ({ isResilient }) =>
-  isResilient ? (
+const DataResiliencyStatusBody: React.FC<DataResiliencyStatusBody> = (error) =>
```

Comment:
destructure the error.
```suggestion
const DataResiliencyStatusBody: React.FC<DataResiliencyStatusBody> = ({error}) =>
```

Reply:
ack